tests/llmobs/_utils.py

```diff
@@ -101,8 +101,24 @@ def _expected_llmobs_llm_span_event(
     meta_dict = {"input": {}, "output": {}}
```

Comment:
can we add some additional test cases to ensure that the message is normalized if no role is specified or if it is set to `None` by the user?

Reply:
added!